### PR TITLE
Increased limits to 58 bits in HighLimit

### PR DESCRIPTION
**Pull request by tinwelint**
and 64 bytes property record size to align with power of two.

### CODE REVIEW

**Comment by chrisvest** [comment]
We can't map files larger than ~2^53 bytes, though.

File: enterprise/kernel/src/test/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimitRecordFormatTest.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 public class HighLimitRecordFormatTest extends RecordFormatTest
 {
-    private static final RecordGenerators HIGH_LIMITS = new LimitedRecordGenerators( random, 50, 50, 50, 16, NULL );
+    private static final RecordGenerators HIGH_LIMITS = new LimitedRecordGenerators( random, 58, 58, 58, 16, NULL );
```
Comment (chrisvest) [inline enterprise/kernel/src/test/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimitRecordFormatTest.java:28]: This is only changing the test. Is that all it takes?

File: enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/PropertyRecordFormat.java [hunk 2]
```diff
@@ -34,7 +34,7 @@
  */
 class PropertyRecordFormat extends BaseOneByteHeaderRecordFormat<PropertyRecord>
 {
-    private static final int RECORD_SIZE = 48;
+    private static final int RECORD_SIZE = 64;
```
Comment (chrisvest) [inline enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/PropertyRecordFormat.java:37]: This doesn't change the `PropertyType.getPayloadSizeLongs()` as far as I can see, so this is just wasting space.

**Comment by tinwelint** [comment]
Some measuring before doing this would be handy

**Pull request closed without merging**